WooCommerce/woocommerce_rms_normal/class-gateway.php

```diff
@@ -1,628 +1,874 @@
+    {
+?>
+        <h3><?php _e('Fiuu', 'wcmolpay'); ?></h3>
+        <p><?php _e('Fiuu works by sending the user to Fiuu to enter their payment information.', 'wcmolpay'); ?></p>
+        <table class="form-table">
+            <?php $this->generate_settings_html(); ?>
+        </table><!--/.form-table-->
+<?php
+    }
+
+    /**
+     * Gateway Settings Form Fields.
+     * 
+     */
+    public function init_form_fields()
+    {
+        $this->form_fields = array(
+            'enabled' => array(
+                'title' => __('Enable/Disable', 'wcmolpay'),
+                'type' => 'checkbox',
+                'label' => __('Enable Fiuu', 'wcmolpay'),
+                'default' => 'yes'
+            ),
+            'ordering_plugin' => array(
+                'title' => __('<p style="color:red;">Installed Ordering Plugins</p>', 'wcmolpay'),
+                'type' => 'select',
+                'label' => __(' ', 'wcmolpay'),
+                'default' => 'Sequential Order Numbers',
+                'options' => array(
+                    '0' => __('Not install any ordering plugin', 'wcmolpay'),
+                    '1' => __('Sequential Order Numbers', 'wcmolpay'),
+                    '2' => __('Sequential Order Numbers Pro', 'wcmolpay')
+                ),
+                'description' => __('Please select correct ordering plugin as it will affect your order result!!', 'wcmolpay'),
+                'desc_tip' => true,
+            ),
+            'title' => array(
+                'title' => __('Title', 'wcmolpay'),
+                'type' => 'text',
+                'description' => __('This controls the title which the user sees during checkout.', 'wcmolpay'),
+                'default' => __('Fiuu', 'wcmolpay'),
+                'desc_tip' => true,
+            ),
+            'payment_title' => array(
+                'title' => __('Payment Title', 'wcmolpay'),
+                'type' => 'checkbox',
+                'label' => __('Showing channel instead of gateway title after payment.'),
+                'description' => __('This controls the payment method which the user sees after payment.', 'wcmolpay'),
+                'default' => 'no',
+                'desc_tip' => true
+            ),
+            'description' => array(
+                'title' => __('Description', 'wcmolpay'),
+                'type' => 'textarea',
+                'description' => __('This controls the description which the user sees during checkout.', 'wcmolpay'),
+                'default' => __('Pay with Fiuu', 'wcmolpay'),
+                'desc_tip' => true,
+            ),
+            'merchant_id' => array(
+                'title' => __('Merchant ID', 'wcmolpay'),
+                'type' => 'text',
+                'description' => __('Please enter your Fiuu Merchant ID.', 'wcmolpay') . ' ' . sprintf(__('You can to get this information in: %sFiuu Account%s.', 'wcmolpay'), '<a href="https://portal.merchant.razer.com/" target="_blank">', '</a>'),
```

Comment:
change to portal.fiuu.com

Reply:
changed 

proof: 013ce67a70abd17bd3027c6e2410e84c1b2c025a